src/zenml/zen_stores/schemas/schema_utils.py

```diff
@@ -0,0 +1,76 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Utility functions for SQLModel schemas."""
+
+from typing import Any, Optional
+
+from sqlalchemy import Column, ForeignKey
+from sqlmodel import Field
+
+
+def foreign_key_constraint_name(
+    source: str, target: str, source_column: str
+) -> str:
+    """Defines the name of a foreign key constraint.
+
+    For simplicity, we use the naming convention used by alembic here:
+    https://alembic.sqlalchemy.org/en/latest/batch.html#dropping-unnamed-or-named-foreign-key-constraints.
+
+    Args:
+        source: Source table name.
+        target: Target table name.
+        source_column: Source column name.
+
+    Returns:
+        Name of the foreign key constraint.
+    """
+    return f"fk_{source}_{source_column}_{target}"
+
+
+def build_foreign_key_field(
+    source: str,
+    target: str,
+    source_column: str,
+    target_column: str,
+    ondelete: Optional[str] = None,
+    **sa_column_kwargs: Any,
+) -> Any:
```

Comment:
Why is this returning `Any` and not values of type `Field`?

Reply:
That's what I wanted to do initially as well. However, `sqlmodel.Field` is actually not a class, it is a function that returns type `Any` 😅